script.py

```diff
@@ -256,14 +256,20 @@ def printLogo():
     )
 
 def main():
+    arguments = Arguments.parse()
+    GLOBAL.arguments = arguments
 
-    if not Path(GLOBAL.defaultConfigDirectory).is_dir():
-        os.makedirs(GLOBAL.defaultConfigDirectory)
-
-    if Path("config.json").exists():
-        GLOBAL.configDirectory = Path("config.json")
+    if arguments.config:
+        if arguments.use_local_config:
```

Comment:
Why does it exit if `--use-local-config` is given?

Reply:
These options inherently conflict with each other, with `--use-local-config` creating a new config if it doesn't exist in current directory, as well as loading it from the current directory.

Specifying `--config` implicitly means that you've already created the config or are loading it from a known location.

Happy to remove the check here. I have no strong opinions either way :)

Reply:
Hrmm, it's been two months since your comment with no responses. I'm going to resolve this.